docs/GettingStarted.md

```diff
@@ -0,0 +1,80 @@
+# Getting started
+
+All you'll a [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and the Hail source code: either clone the [Hail repository](https://github.com/broadinstitute/hail) using [Git](https://git-scm.com/) with
+```
+git clone https://github.com/broadinstitute/hail.git
+```
+or download the source code directly from [here](https://github.com/broadinstitute/hail/archive/master.zip), unzip, and rename the enclosing `hail-master` directory to `hail`. The following commands are relative to this `hail` directory.
+
+## Building and running Hail
+
+Hail may be built to run locally with all cores on a single machine or to run on a Spark cluster. Running locally is useful for getting started, analyzing or experimenting with small datasets, and Hail development.
+
+### Building and running Hail locally
+The single command
+```
+./gradlew installDist
+```
+will populate `build/install` with an installation of Hail, including the executible `build/install/hail/bin/hail`. The initial build takes time as the build tool [Gradle](https://gradle.org/) installs and then downloads all Hail build dependencies, including the Scala compiler and Spark.
+
+Here are a few simple things to try. To list all commands, run
+```
+./build/install/hail/bin/hail -h
+```
+To convert the included `sample.vcf` to Hail's `.vds` format, run
+```
+./build/install/hail/bin/hail importvcf src/test/resources/sample.vcf write -o ~/sample.vds
+```
+Then to count the number of samples and variants, run
+```
+./build/install/hail/bin/hail read -i ~/sample.vds count
+```
+To compute and output sample and variant quality control statistics, run
+```
+./build/install/hail/bin/hail read -i ~/sample.vds variantqc -o ~/variantqc.tsv sampleqc -o ~/sampleqc.tsv
+```
+To run Hail using `hail` in place of `./build/install/hail/bin/hail`, add this `bin` directory to your path. Note that during each run Hail writes a `hail.log` file in the current directory; this is useful to developers for debugging.
+
+### Building a Spark application JAR
+
+Hail is compatible with Spark versions 1.5 and 1.6, and uses 1.5.2 by default. Set the Spark version to match that of the Spark cluster with
+```
+./gradlew -Dspark.version=VERSION
+```
+Then run
+```
+./gradlew shadowJar
+```
+to build `build/libs/hail-all-spark.jar`, which can be submitted using `spark-submit` (see [Spark documentation](http://spark.apache.org/docs/1.6.2/submitting-applications.html) for details).
+
+### BLAS and LAPACK
+
+Hail uses the Scala numerical processing library [Breeze](http://github.com/scalanlp/breeze), which in turn uses the [netlib-java](http://github.com/fommil/netlib-java) to take advantage of native [BLAS](http://www.netlib.org/blas/) and [LAPACK](http://www.netlib.org/lapack/) routines. These routines are available by default on OS X and Windows. On Linux, they must be explicitly enabled. On Ubuntu 14.04, run
```

Comment:
Detail overload.  Just say "Hail uses BLAS and LAPACK optimized linear algebra [or numeric processing] libraries."

Say installed, not enabled.

BLAS and LAPACK are available by default on Windows?  To my knowledge, nobody has tried to build or run Hail on Windows.  I think you shouldn't mention it.